medusa/subtitle_providers/legendastv.py

```diff
@@ -457,7 +457,14 @@ def list_subtitles(self, video, languages):
         else:
             title = video.title
 
-        return [s for l in languages for s in self.query(l, title, season=season, episode=episode, year=video.year)]
+        found_subtitles = [s for l in languages for s in self.query(l, title, season=season, episode=episode, year=video.year)]
+        if not found_subtitles and 'marve' in title.lower():
+            title = title.lower().replace("marvel's ", '').replace('marvels ', '')
+            found_subtitles = [s for l in languages for s in self.query(l, title, season=season, episode=episode, year=video.year)]
+        if not found_subtitles and 'legends of tomorrow' in title.lower():
+            title = 'dc’s legends of tomorrow'
+            found_subtitles = [s for l in languages for s in self.query(l, title, season=season, episode=episode, year=video.year)]
+        return found_subtitles
```

Comment:
@ratoaq2 the problem here is that show was inserted in ltv database with this `’`  and not `'`

maybe to fix for other shows we could replace `’`  with `'`